symphony/app/fbcnms-projects/magmalte/app/views/equipment/GatewayDetailEnodebs.js

```diff
@@ -30,7 +30,7 @@ export default function GatewayDetailEnodebs(
   const {history, match} = useRouter();
   const [currRow, setCurrRow] = useState<EnodebRowType>({});
 
-  const enbRows: Array<EnodebRowType> = gwInfo.connected_enodeb_serials.map(
+  const enbRows: Array<EnodebRowType> = gwInfo.connected_enodeb_serials?.map(
     (serialNum: string) => {
```

Comment:
One other issue is the props for this component is buggy. It perhaps should be this. enbInfo was undefined in the current code
{
  gwInfo,
  enbInfo,
}: {
  gwInfo: lte_gateway,
  enbInfo: {[string]: EnodebInfo},
}

I was fixing it in another commit. But i guess it might be better for you to just fix all enodeb issues in this pull request
https://github.com/facebookincubator/magma/pull/1876/commits/6fed31bde95295b1b4ccfed754ceef82f73274eb

Reply:
Yes thanks